templates/index.html

```diff
@@ -23,7 +23,15 @@ <h2>Items</h2>
     <label for="title">title:</label>
       <input type="text" id="title" name="title">
     </li>
-    <button type="submit">Send your title</button>
+    <button type="submit">Add Card</button>
   
    </form>
+
+   <form action="/move_Item" method="post">
+
+  
+    <button type="submit">Move Card</button>
+  
+   </form>
```

Comment:
"Move card" is misleading - it moves all cards from the todo list to the done list, right?

We did want a "move card" form though (see my other comment). To do this you can put it inside the if loop, and include an `<input type="hidden" value="..." name="...">` element in the form for the card id.

Reply:
Please check the changes with commit ID - ad8f0c427b592c542b116bdb03e61ebd8ac26d80